pkg/proxy/ipvs/proxier.go

```diff
@@ -1273,9 +1276,13 @@ func (proxier *Proxier) syncProxyRules() {
 			if proxier.ipvsScheduler == "mh" {
 				serv.Flags |= utilipvs.FlagSourceHash
 			}
-			if err := proxier.syncService(svcPortNameString, serv, true, alreadyBoundAddrs); err == nil {
+			// We must not add the address to the dummy device if it exist on another interface
```

Comment:
I'm not familiar with the ipvs proxy, is it creating a new IP address on this dummy interface for the virtualips that doesn't exist on the node?

Reply:
there is another dimension, GCP loadbalancers insert a route to the VIP like this
```
local 34.79.233.109 dev eth0 table local proto 66 scope host
```

```
ip route get 34.79.233.109
local 34.79.233.109 dev lo src 34.79.233.109 uid 20162
    cache <local>
```

this VIP doesn't seem to be associated to any interface
```
 $ ip add | grep 34
 $
```

@uablrek with this changes, I understand that the IPs that are in a Service and within the node will work, but what will happen with these IPs that are in the Service `{.status.loadBalancer.ingress[0].ip}` but not in an interface?

Reply:
I'm talking about this https://github.com/kubernetes/kubernetes/issues/117621, is this going to fix this or do we need an additional PR?